docs(window): Document ContainmentSelector and revamp Overview and Position articles (#1995)
telerik/blazor-docs: components/window/overview.md

## Window Parameters

The following table lists the Window parameters. Also check the [Window API](/blazor-ui/api/Telerik.Blazor.Components.TelerikWindow) for a full list of parameters, methods, and events.

@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)

| Parameter | Type and Default&nbsp;Value | Description |
| --- | --- | --- |
| `Centered` | `string` <br /> (`true`) | Determines if the Window displays in the middle of the viewport. This parameter is ignored if `Top` or `Left` is set to a non-empty string. |
| `Class` | `string` | The custom CSS class of the `<div class="k-window">` element. Use it to [override theme styles]({%slug themes-override%}). Here is a [custom Window styling example]({%slug window-kb-custom-css-styling%}). |
| `CloseOnOverlayClick` | `bool` | Sets if a modal Window will close when the user clicks on the modal overlay that covers the rest of the page content. |

> | `ContainmentSelector` | `string` | A CSS selector that points to a unique HTML element on the page. The Window will render inside 's resizing and dragging will be restricted to the boundaries of the specified container, so the the component visually inside. Do not use with modal Windows. |

Author: ```suggestion
| `ContainmentSelector` | `string` | A CSS selector that points to a unique HTML element on the page. The Window will render inside the specified container. Window resizing and dragging will be restricted by the boundaries of the specified container. Do not use `ContainmentSelector` with modal Windows. |
```

| --- | --- |
| `Refresh` | Redraws the component. |

>caption Get a reference to the Window and use its Refresh method.

````CSHTML
<TelerikButton OnClick="@OpenWindow">Open Window</TelerikButton>

<TelerikWindow @ref="WindowRef" @bind-Visible="@WindowVisible">
    <WindowTitle>
        Window Title
    </WindowTitle>
    <WindowActions>
        <WindowAction Name="Close" />
    </WindowActions>
    <WindowContent>
        <p role="status">Current count: @CurrentCount</p>
        <TelerikButton OnClick="IncrementCount">Increment Count</TelerikButton>
    </WindowContent>
</TelerikWindow>

@code {
    private TelerikWindow? WindowRef { get; set; }

    private bool WindowVisible { get; set; }

    private int CurrentCount { get; set; }

    private void IncrementCount()
    {
        CurrentCount++;

        WindowRef?.Refresh();
    }

    private void OpenWindow()
    {
        WindowVisible = true;
    }
}
````

## Important Notes

> When [`ContainmentSelector`]({%slug components/window/position%}#containmentselector) is not set, the Telerik Window component renders as a child of the [`TelerikRootComponent`]({%slug rootcomponent-overview%}) at the root of the Blazor app. This placement ensures that the Window can show over all the other page content in all scenarios, and have a correct position.

Reviewer: ```suggestion
When [`ContainmentSelector`]({%slug components/window/position%}#containmentselector) is not set, the Telerik Window component renders as a child of the [`TelerikRootComponent`]({%slug rootcomponent-overview%}) at the root of the Blazor app. This placement ensures that the Window can show over all the other page content in all scenarios and have a correct position.
```
